packages/store/src/secure-store/index.ts

```diff
@@ -12,6 +15,23 @@ function getKeytar(log: Logger): typeof Keytar | undefined {
         log.warn(errorString(err));
         log.warn(`Could not "require('keytar')". Trying VSCode's copy`);
```

Comment:
This message should be below the new code.

Reply:
Thank you, applied ✅